app/services/catalog_search_service.rb

```diff
@@ -7,6 +7,9 @@ def initialize(options = {})
   end
 
   class Response < AbstractSearchService::Response
+    HIGHLIGHTED_FACET_FIELD = 'format_main_ssim'.freeze
```

Comment:
I suppose this (and the article facet field) could be be in settings.yml.  I don't feel strongly about it so happy to change it.

Reply:
/whatev's